tools/create_repository.py

```diff
@@ -335,10 +335,7 @@ def create_repository(
         info_file.write(info_contents)
 
     # Calculate the signature.
-    digest = hashlib.md5(info_contents).hexdigest()
-    with open(checksum_path, 'w') as sig:
-        sig.write(digest)
-
+    generate_checksum(info_path)
```

Comment:
Looks like I remembered this wrong. I don't need info_contents. I'll definitely be keeping your change to this part of the file.